Microsoft.Azure.Cosmos/src/ChangeFeedProcessor/ChangeFeedEstimatorIterator.cs

```diff
@@ -279,6 +280,22 @@ private static IEnumerable<JObject> GetItemsFromResponse(ResponseMessage respons
             try
             {
                 ResponseMessage response = await iterator.ReadNextAsync(trace, cancellationToken).ConfigureAwait(false);
+
+                // NOTE(philipthomas-MSFT):
+                // https://github.com/Azure/azure-cosmos-dotnet-v3/issues/4285
+                // Given a change feed processor has started, then stopped.
+                //     And the change feed processor is now dormant.
+                // When a split occurs on the monitored container.
+                //     And the change feed estimator has started.
+                //     And it is using the same lease container before the split occured.
+                //     And a ReadNextAsync returns a 410/1002 due to the partition no longer existing.
+                // Then the state of the change feed processor will have an estimatedLag equal to '1'.
+
+                if (response.StatusCode == HttpStatusCode.Gone && response.Headers.SubStatusCode == SubStatusCodes.PartitionKeyRangeGone)
```

Comment:
```suggestion
                if (response.StatusCode == HttpStatusCode.Gone && response.Headers.SubStatusCode == Documents.SubStatusCodes.PartitionKeyRangeGone)
```

This file is already using the `Documents.` prefix to avoid the `using` on the top. You can remove the `using` just by adding this prefix to the line.

Reply:
done